include/ajax.tasks.php

```diff
@@ -424,7 +422,7 @@ function massProcess($action, $w=null)  {
 
             // Assume success
             if ($i==$count) {
-                $msg = sprintf(__('Successfully %s %s.'),
+                $msg = sprintf(__('Successfully %1$s %2$s.' /* Tokens are <actioned> <x selected task(s)> */),
```

Comment:
Sure I could do so, but... :D
Basically I just copied that comment style from Jared. I think he did it this way because of the translations over at crowdin. Without the comment, the translators often asked for what '%1$S' and '%2$s' are standing for, in order to correctly translate such phrases.
To avoid that I guess Jared added these comments. Maybe there's a better way to satisfy both sides (devs and translators)?